packages/common/test-utils.js

```diff
@@ -10,6 +10,15 @@ const fs = require('fs-extra');
 
 exports.inTestMode = () => process.env.NODE_ENV === 'test';
 
+/**
```

Comment:
This seems like a better compromise for unit test exported functions, absent other solutions (e.g. TestDouble) in situations like this PR where mocking necessitates what would otherwise be an unexported object.